### PR TITLE
black-primer: handle singular and plural in output messages

### PR DESCRIPTION
**Pull request by hugovk**
Done in a similar way to the main Black code, for example:

```python
s = "" if results.stats["disabled"] == 1 else "s"
click.echo(f" - {results.stats['disabled']} project{s} disabled by config")
```

Also some typos and consistency.

### CODE REVIEW

**Comment by cooperlees** [comment]
One Travis passes so going to merge - Can’t find the restart button on Travis.

File: src/black_primer/cli.py [hunk 1]
```diff
@@ -121,9 +121,9 @@ async def async_main(
 )
 @click.pass_context
 def main(ctx: click.core.Context, **kwargs: Any) -> None:
-    """primer - prime projects for blackening ... 🏴"""
+    """primer - prime projects for blackening... 🏴"""
```
[the remark this answers is not in the document]
Comment (hugovk) [reply src/black_primer/cli.py:124]: It's a minor change for sure, and achieves consistency with the rest of the codebase, and uses the most common form.

**Comment by hugovk** [comment]
> Can’t find the restart button on Travis.

I had the same thing on another repo recently, logging out of Travis CI and back in  sorted it.

**Comment by cooperlees** [comment]
Handy to know. Thanks.

**Pull request merged**